src/Equinox.MessageDb/MessageDbClient.fs

```diff
@@ -64,16 +65,13 @@ type MessageDbReader internal (connectionString : string, leaderConnectionString
 
     let parseRow (reader : DbDataReader) : ITimelineEvent<Format> =
         let inline readNullableString idx = if reader.IsDBNull(idx) then None else Some (reader.GetString idx)
-
-        TimelineEvent.Create(
+        let c, d, m = reader.GetString(1), reader |> Json.fromReader 2, reader |> Json.fromReader 3
```

Comment:
Not sure this is an improvement readability wise 🤔 can we keep it as `eventType`, `data`, and `meta`?

Reply:
Not a blocker at all, can see the argument for keeping it unified across the stores as well

Reply:
that was what I was driving at, and I do realise that we can afford the space
the c for case is a bit ugly/debatable, but it prepares people for the other stores
lets see how I feel after my run!

Reply:
Problem solved, and I saw a stupid bug I had introduced